sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/display/DisplayDataTest.java

```diff
@@ -165,6 +167,51 @@ public void populateDisplayData(DisplayData.Builder builder) {
   }
 
```

Comment:
Is there support for late-bound `PipelineOptions` to appear as display data in the `PipelineOptions`? Currently, display data is JSON serialized as part of `PipelineOptions` ([here](https://github.com/apache/incubator-beam/blob/2c0d0f476ba92bf3b18bcc96f3dac47da176575d/sdks/java/core/src/main/java/org/apache/beam/sdk/options/ProxyInvocationHandler.java#L626), and [here in Dataflow](https://github.com/apache/incubator-beam/blob/2c0d0f476ba92bf3b18bcc96f3dac47da176575d/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslator.java#L416)).

I'm not familiar with how template jobs get bound and run, but as long the entire `PipelineOptions` JSON is replaced with bound values, the DisplayData portion should get populated as well.

If there's not a test for this already, can you add one?